File: u3u2/Assets/Scripts/app/net/c2s/CGGiveUpTlNpc.cs
using System;
using System.IO;
namespace app.net
{

/**
 * 放弃已接任务
 * 
 * @author CodeGenerator, don't modify this file please.
 */
public class CGGiveUpTlNpc :BaseMessage
{
	
	
	public CGGiveUpTlNpc ()
	{
	}
	
	
	protected override void ReadImpl() 
	{
		return;
	}
	
	protected override void WriteImpl() 
	{

	}
	
	public override short GetMessageType()
	{
		return (short)MessageType.CG_GIVE_UP_TL_NPC;
	}
	
	public override string getEventType()
	{
		return "";
	}
	
	}
}

File: server/当前版本/server/zlj/game_server/src/com/imop/lj/gameserver/timelimit/msg/CGGiveUpTlNpc.java
package com.imop.lj.gameserver.timelimit.msg;

import com.imop.lj.core.msg.MessageType;
import com.imop.lj.gameserver.common.msg.CGMessage;
import com.imop.lj.gameserver.timelimit.handler.TimelimitHandlerFactory;

/**
 * 放弃已接任务
 * 
 * @author CodeGenerator, don't modify this file please.
 */
public class CGGiveUpTlNpc extends CGMessage{
	
	
	public CGGiveUpTlNpc (){
	}
	
	
	@Override
	protected boolean readImpl() {


		return true;
	}
	
	@Override
	protected boolean writeImpl() {

		return true;
	}
	
	@Override
	public short getType() {
		return MessageType.CG_GIVE_UP_TL_NPC;
	}
	
	@Override
	public String getTypeName() {
		return "CG_GIVE_UP_TL_NPC";
	}


	@Override
	public void execute() {
		TimelimitHandlerFactory.getHandler().handleGiveUpTlNpc(this.getSession().getPlayer(), this);
	}
}